docs/0-quickstart.ipynb

```diff
@@ -0,0 +1,489 @@
+    "1. define model frame\n",
+    "2. construct observation\n",
+    "3. match it to the model frame\n",
+    "\n",
+    "Steps 2 and 3 are combined above using a fluent pattern.\n",
+    "\n",
+    "## Initialize sources\n",
+    "\n",
+    "You now need to define sources that are going to be fit. The full model, which we will call `Blend`, is a collection of those sources. We provide several pre-built source types:\n",
+    "\n",
+    "* `RandomSource` fit per-band amplitude and non-parametric morphology starting from uniform random draws for both.\n",
+    "* `PointSource` fits centers and per-band amplitude using the observed PSF model.\n",
+    "* `ExtendedSource` fits per-band amplitude and a non-parametric morphology (which can be constrained to be symmetric and/or monotonic with respect to the center).\n",
+    "* `MultiComponentSource` splits an `ExtendedSource` into multiple components that are initially radially separated.\n",
+    "\n",
+    "In our example, we assume *prior* knowledge that object 0 is a star, and object 1 should be modeled with two components. Everything else is assumed a single-component galaxy."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sources = []\n",
+    "for k,src in enumerate(catalog):\n",
+    "    if k == 0:\n",
+    "        new_source = scarlet.PointSource(model_frame, (src['y'], src['x']), observation)\n",
+    "    elif k == 1:\n",
+    "        new_source = scarlet.MultiComponentSource(model_frame, (src['y'], src['x']), observation, symmetric=False, monotonic=True, thresh=5)\n",
+    "    else:\n",
+    "        new_source = scarlet.ExtendedSource(model_frame, (src['y'], src['x']), observation, symmetric=False, monotonic=True, thresh=5)\n",
+    "    sources.append(new_source)"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    ".. warning::\n",
+    "\n",
+    "    Coordinates in *scarlet* are given in the C/numpy notation (y,x) as opposed to the more conventional mathematical (x,y) ordering."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create and fit model\n",
+    "The `Blend` class represent the sources as a tree and has the machinery to fit all of the sources to the given images. In this example the code is set to run for a maximum of 200 iterations, but will end early if the likelihood and all of the constraints converge."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "blend = scarlet.Blend(sources, observation)\n",
+    "%time blend.fit(200)\n",
+    "print(\"scarlet ran for {0} iterations to logL = {1}\".format(len(blend.loss), -blend.loss[-1]))\n",
+    "plt.plot(-np.array(blend.loss))\n",
+    "plt.xlabel('Iteration')\n",
+    "plt.ylabel('log-Likelihood')"
+   ]
```

Comment:
This is a `proxmin` message, which only checks for parameter convergence since it doesn't have access to the function value. `scarlet` uses an additional functional convergence criterion, which often kicks in earlier. Open for suggestions in dealing with these disparate log messages.

Reply:
Ok, I'll have to look back at proxmin to understand the difference and where/why it is being flagged in the first place.

Reply:
These are two different criteria. In proxmin, we only have parameter convergence because it never evaluates the loss function, it only works with its gradients. In scarlet, we get the loss function value for free when computing the gradients with autograd, so I've added this extra convergence check. In fact, I think it's more meaningful since that's the function we seek to optimize, and non-convergence of faint objects doesn't interfere with that. Robert argued the same. As it stands, the proxmin non-convergence complaint is equivalent to "There are faint sources in the blend". I propose that we silence it.

Reply:
I agree with silencing this warning in proxmin.

Reply:
Streamlined the logging. `Blend` now silences warnings from proxmin, but than can be overwritten by the user is necessary.